The script to generate 24px versions of the icons uses GNUisms when calling grep and will thus not work with Busybox's grep.
This will be upstreamed when I have time for it.

diff --git a/generate-24px-versions.sh b/generate-24px-versions.sh
index 470c755..4d73a83 100644
--- a/generate-24px-versions.sh
+++ b/generate-24px-versions.sh
@@ -36,10 +36,10 @@ for INPUT_FILE in {devices,actions,places,status}/22/*.svg; do
         cp "$INPUT_FILE" "$OUTPUT_FILE"
 
         # Change the size of the bounding box
-        if grep --quiet '"0 0 22 22' "$OUTPUT_FILE"; then
+        if grep -q '"0 0 22 22' "$OUTPUT_FILE"; then
             sed_i -e 's/"0 0 22 22/"0 0 24 24/g' "$OUTPUT_FILE"
         # YOU
-        elif grep --quiet 'width="22"' "$OUTPUT_FILE"; then
+        elif grep -q 'width="22"' "$OUTPUT_FILE"; then
             sed_i -e 's/width="22"/width="24"/g' "$OUTPUT_FILE"
             sed_i -e 's/height="22"/height="24"/g' "$OUTPUT_FILE"
         else
@@ -48,9 +48,9 @@ for INPUT_FILE in {devices,actions,places,status}/22/*.svg; do
         fi
 
         # Group all the paths and transform them to fit in the center of a 24px box
-        if grep --quiet "</defs>" "$OUTPUT_FILE"; then
+        if grep -q "</defs>" "$OUTPUT_FILE"; then
             sed_i -e 's|</defs>|</defs><g transform="translate(1,1)">|g' "$OUTPUT_FILE"
-        elif grep --quiet "</style>" "$OUTPUT_FILE"; then
+        elif grep -q "</style>" "$OUTPUT_FILE"; then
             sed_i -e 's|</style>|</style><g transform="translate(1,1)">|g' "$OUTPUT_FILE"
         else
         # No stylesheet or defs tag; put the group at the end of the opening svg tag
